Introduce MultiCommand class for cmds operating on multiple todos.

mruwek opened the pull request:
This fixes #14

@bram85 I've implemented your suggestions and added new `execute_multi_specific()` method to reduce indentation and overall code duplication in `execute()`. Tests are still valid :)

bram85 commented on topydo/lib/MultiCommand.py:
```diff
@@ -0,0 +1,68 @@
+            except InvalidTodoException:
+                self.invalid_numbers.append(number)
+
+    def catch_todo_errors(self):
```
Function should be private (start with `_`).

bram85 replied:
Please also add a docstring here.

bram85 commented on topydo/lib/MultiCommand.py:
```diff
@@ -0,0 +1,68 @@
+        self.todos = []
+        self.invalid_numbers = []
+
+    def get_todos(self, p_numbers):
```
A small docstring that describes what this function does would be helpful.

bram85 commented:
Looks good overall, will merge it in if you deal with the minor remarks (doesn't affect any functionality, just style things).

mruwek commented:
Updated and rebased. Changelog:
1. Add docstrings for `MultiCommand.catch_errors()` and `MultiCommand.get_todos()`
2. Turn `MultiCommand.catch_errors()` into private method --> `MultiCommand._catch_errors()`
3. Define `MultiCommand.execute_multi_specific()` in `MultiCommand.MultiCommand` itself and add docstring there.

bram85 commented:
Thanks for doing this.
